test/support/utils_invoke.h

```diff
@@ -37,6 +37,52 @@ struct unique_kernel_name;
 template <typename Policy, int idx>
 using new_kernel_name = unique_kernel_name<typename ::std::decay<Policy>::type, idx>;
 
+constexpr bool
+use_explicit_kernel_names()
+{
+#ifndef TEST_EXPLICIT_KERNEL_NAMES
+    static_assert(false, "The state of TEST_EXPLICIT_KERNEL_NAMES is not defined");
+#endif
+
+    return TEST_EXPLICIT_KERNEL_NAMES ? true : false;
+}
+
+/**
+ * make_policy functions test wrappers
+ * The main purpose of this function wrapper in TestUtils namespace - to cut template params from
+ * oneapi::dpl::execution::device_policy function calls depends on TEST_EXPLICIT_KERNEL_NAMES macro state.
+ * 
+ * ATTENTION: Please avoid to use oneapi::dpl::execution::device_policy directly in tests.
+ */
+template <typename KernelName = oneapi::dpl::execution::DefaultKernelName, typename Arg>
+inline auto
+make_device_policy(Arg&& arg)
+{
+    if constexpr (use_explicit_kernel_names())
+        return oneapi::dpl::execution::make_device_policy<KernelName>(::std::forward<Arg>(arg));
+    else
+        return oneapi::dpl::execution::make_device_policy(::std::forward<Arg>(arg));
+}
+
+#if _ONEDPL_FPGA_DEVICE
+/**
+ * make_fpga_policy functions test wrappers
+ * The main purpose of this function wrapper in TestUtils namespace - to cut template params from
+ * oneapi::dpl::execution::device_policy function calls depends on TEST_EXPLICIT_KERNEL_NAMES macro state.
+ * 
+ * ATTENTION: Please avoid to use oneapi::dpl::execution::make_fpga_policy directly in tests.
+ */
+template <unsigned int new_unroll_factor, typename NewKernelName, typename Arg>
```

Comment:
Should it be aligned with the interface above?
```suggestion
template <unsigned int new_unroll_factor = 1, typename NewKernelName = DefaultKernelNameFPGA, typename Arg>
```

Reply:
Fixed (applied)